src/sentry/assistant/guides.py

```diff
@@ -150,5 +170,54 @@
                 'target': 'member_role',
             },
         ]
-    }
+    },
+    # Ideally, this would only be sent if the organization has never
+    # customized alert rules (as per FeatureAdoption)
+    'alert_rules': {
+        'id': 5,
+        'cue': _('Tips for alert rules'),
+        'required_targets': ['alert_conditions'],
+        'steps': [
+            {
+                'title': _('Reduce inbox noise'),
+                'message': _('Sentry, by default, alerts on every <i>new</i> issue via email. '
+                             'If that\'s too noisy, sending the new issue alerts to '
+                             'a service like Slack help reduce inbox noise.<br><br> Enabling '
+                             '<a href="https://sentry.io/settings/account/notifications/#weeklyReports">'
```

Comment:
add target="_blank" to all links (we don't want them to navigate away from the page in the middle of a tour)

Reply:
Good point.